src/main/scala/mesosphere/marathon/api/v2/AppsResource.scala

```diff
@@ -315,34 +323,3 @@ class AppsResource @Inject() (
     }
   }
 }
-
-object AppsResource {
-  implicit val WithTaskCountsAndDeploymentsWrites: Writes[AppDefinition.WithTaskCountsAndDeployments] = Writes { app =>
-    val appJson = AppDefinitionWrites.writes(app).as[JsObject]
-
-    appJson ++ Json.obj(
-      "tasksStaged" -> app.tasksStaged,
-      "tasksRunning" -> app.tasksRunning,
-      "tasksHealthy" -> app.tasksHealthy,
-      "tasksUnhealthy" -> app.tasksUnhealthy,
```

Comment:
Generally, I prefer this explicit conversion to the stuff using JsonProperties. I am bit confused about the pre-existing @JsonProperty methods, though. Was that redundant code here?